cgi-bin/LJ/Entry.pm

```diff
@@ -2045,6 +2045,70 @@ sub load_log_props2
 }
 
 # <LJFUNC>
+# name: LJ::load_talk_props2
```

Comment:
The problem I saw with using LJ::Comment is that it doesn't already include any functions in package LJ, whereas LJ::Entry already includes package LJ functions for both entries and comments (see LJ::delete_all_comments, LJ::delete_comments).  Also LJ::Entry is already included in all the places where the functions being moved are called.  It's much easier and more consistent to do it this way, in my view.